schema/me/save_artwork.js

```diff
@@ -0,0 +1,28 @@
+import gravity from '../../lib/loaders/gravity';
+import { GraphQLString, GraphQLBoolean } from 'graphql';
+import { mutationWithClientMutationId } from 'graphql-relay';
+import { artworkFields } from '../artwork/index';
+
+export default mutationWithClientMutationId({
+  name: 'SaveArtwork',
+  decription: 'Save (or remove) an artwork to (from) a users default collection.',
+  inputFields: {
+    artwork_id: {
+      type: GraphQLString,
+    },
+    remove: {
+      type: GraphQLBoolean,
+    },
+  },
+  outputFields: artworkFields(),
```

Comment:
Yeah, because of the whole using the Artwork.type within the artwork type definition